npm/pkg/controlplane/translation/parseSelector.go

```diff
@@ -245,7 +245,11 @@ func parsePodSelector(selector *metav1.LabelSelector) []labelSelector {
 		var setName string
 		var setType ipsets.SetType
 		var members []string
-		switch op := req.Operator; op {
+		op := req.Operator
+		if invalidWindowsOperatorLimitation(op) {
```

Comment:
Nitpick: slightly better `unsupportedOpsInWindows` instead of `invalidWindowsOperatorLimitation` 
no strong opinion.